test/integration/controller/core/clusterqueue_controller_test.go

```diff
@@ -173,8 +205,16 @@ var _ = ginkgo.Describe("ClusterQueue controller", func() {
 						},
 					},
 				},
-			}))
-			framework.ExpectPendingWorkloadsMetric(clusterQueue, 0, 1)
+				Conditions: []metav1.Condition{
+					{
+						Type:    kueue.ClusterQueueActive,
+						Status:  metav1.ConditionTrue,
+						Reason:  "Ready",
+						Message: "Can admit new workloads",
+					},
+				},
+			}, ignoreCQConditionTimestamps))
+			framework.ExpectPendingWorkloadsMetric(clusterQueue, 1, 0)
```

Comment:
Since ResourceFlavors are created above, the inadmissionPending is expected to be `0`, and the activePending is expected to be `1`.